src/docs/index.md

```diff
@@ -128,6 +128,96 @@ title: Akka.NET Documentation
 - [Scheduler](Scheduler)
 - [Circuit Breaker](CircuitBreaker)
 
+### Akka Streams (work in progress)
+*we are currently in the progress of porting the streams documentation for Akka.Net. As such not all sections will be available*
+
+- [Introduction](streams/introduction)
+	- [Motivation](streams/introduction#motivation)
+	- [How to read these docs](streams/introduction#how-to-read-these-docs)
+- [Quick Start Guide](streams/quickstart)
+	- [Reusable Piece](streams/quickstart#reusable-pieces)
+	- [Time-Based Processing](streams/quickstart#time-based-processing)
+- [Reactive Tweets](streams/reactivetweets)
+	- [Transforming and consuming simple streams](streams/reactivetweets#transforming-and-consuming-simple-streams)
+	- [Flattening sequences in streams](streams/reactivetweets#flattening-sequences-in-streams)
+	- [Broadcasting a stream](streams/reactivetweets#broadcasting-a-stream)
+	- [Back-pressure in action](streams/reactivetweets#back-pressure-in-action)
+	- [Materialized values](streams/reactivetweets#materialized-values)
+- [Design Principles behind Akka Streams](streams/designprinciples)
+	- [What shall users of Akka Streams expect?](streams/designprinciples#what-shall-users-of-akka-streams-expect)
+	- [Interoperation with other Reactive Streams implementations](streams/designprinciples#interoperation-with-other-reactive-streams-implemenations)
```

Comment:
There are no other implementations of reactive streams in .NET space, and Akka.NET streams probably won't interop with JVM implementations just yet.